pom.xml

```diff
@@ -62,6 +62,13 @@
 
   <dependencies>
     <!-- regular dependencies -->
+    <dependency>
+      <groupId>org.jenkins-ci</groupId>
+      <artifactId>trilead-ssh2</artifactId>
+      <version>build217-jenkins-9</version>
```

Comment:
Nope. We want the oldest version of Trilead that provides the new API. This version is only used during the compile/test phase, not in the package/release phase so the actual version here doesn't impact what end users get, but it needs to be a version that provides the API required in the PR without introducing other API that may not exist in versions bundled in other Jenkins releases.